tests/RunTests.cmd

```diff
@@ -172,18 +172,18 @@ IF NOT DEFINED GACUTILEXE64 IF EXIST "%WINSDKNETFXTOOLS%x64\gacutil.exe" set GAC
 set FSC=%FSCBINPATH%\fsc.exe
 set PATH=%FSCBINPATH%;%PATH%
 
-set FSCVPREVBINPATH=%X86_PROGRAMFILES%\Microsoft SDKs\F#\3.1\Framework\v4.0
+set FSCVPREVBINPATH=%X86_PROGRAMFILES%\Microsoft SDKs\F#\4.0\Framework\v4.0
```

Comment:
I think this is used to test consumption of binaries produced with previous editions of F#, and to check that previous editions of F# can consume binaries produced with the current edition when referencing 4.3.0.0, 4.3.1.0 and 4.4.0.0

Because his is about backwards-forwards compat, I think we shouldn't be dropping F# 3.1 testing here but rather augmenting it with F# 4.0 testing as well